packages/checkout-ui-extensions/src/components/Text/README.md

```diff
@@ -9,8 +9,8 @@ optional = ?
 | Name | Type | Description |
 | --- | --- | --- |
 | size? | <code>"extraSmall" &#124; "small" &#124; "base" &#124; "large" &#124; "extraLarge" &#124; "medium"</code> | Size of the text |
-| emphasis? | <code>"stress" &#124; "offset" &#124; "strong"</code> | Use to emphasize a word or a group of words compared to other nearby text.<br /><br /><code>"stress"</code>: Indicate emphatic stress. By default the text will be italic and render a `<em>` tag in an HTML host.<br /><br /><code>"offset"</code>: Indicate an offset from the normal prose of the text. Typically used to indicate a foreign word, fictional character thoughts, or when the text refers to the definition of a word instead of representing its semantic meaning. By default the text will be italic and render a `<i>` tag in an HTML host.<br /><br /><code>"strong"</code>: Indicate strong importance, seriousness, or urgency. By default the text will be bold and render a `<strong>` tag in an HTML host. |
-| accessibilityRole? | <code>"address" &#124; "deletion" &#124; "marking" &#124; {type: "abbreviation", for?: string} &#124; {type: "directional-override", direction: "ltr" &#124; "rtl"} &#124; {type: "datetime", machineReadable?: string}</code> | Set the semantic of the component’s content<br /><br /><code>"address"</code>: Indicate the text is contact information. Typically used for addresses.<br /><br /><code>"deletion"</code>: Indicate the text has been deleted. Typically used for discounted prices.<br /><br /><code>"marking"</code>: Indicate the text is marked or highlighted and relevant to the buyer's current action. Typically used to indicate the characters that matched a search query.<br /><br /><code>{type: "abbreviation", for?: string}</code>: Indicate the text is an abbreviation or acronym. Use the “for” option to provide a description of the abbreviation.<br /><br /><code>{type: "directional-override", direction: "ltr" &#124; "rtl"}</code>: Override the text directionality. Typically used for email and phone numbers.<br /><br /><code>{type: "datetime", machineReadable?: string}</code>: Indicate the text is a date, a time or a duration. Use the “machineReadable” option to help browsers, tools or software understand the human-readable date. The valid format for “machineReadable” can be found here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time#Valid_datetime_Values |
+| emphasis? | <code>"italic" &#124; "bold"</code> | Use to emphasize a word or a group of words compared to other nearby text.<br /><br /><code>"italic"</code>: Set the text in italic. Combine with an accessibilyRole of 'offset' or 'stress' to add more meaning to the text.<br /><br /><code>"bold"</code>: Set the text in bold. |
```

Comment:
```suggestion
| emphasis? | <code>"italic" &#124; "bold"</code> | Use to emphasize a word or a group of words compared to other nearby text.<br /><br /><code>"italic"</code>: Set the text in italic. Combine with an “accessibilyRole“ of “offset“ or “stress“ to add more meaning to the text.<br /><br /><code>"bold"</code>: Set the text in bold. Combine with an “accessibilyRole“ of “strong“ to add more meaning to the text. |
```

Reply:
Shouldn't we wrap accessibilyRole, offset and stress in backticks (`) to make them render like code?